regression-test/suites/nereids_p0/stats/partition_col_stats.groovy

```diff
@@ -36,10 +36,18 @@ suite("partition_col_stats") {
     """
     //run this sql to make stats be cached
     sql "select * from pt where k1<3;"
-    sleep(10000)
+    def pt_data = sql "show data from pt;"
```

Comment:
should set nonConcurrent for this suite, refer to `regression-test/suites/meta_action_p0/test_dump_image.groovy`

```
suite("test_dump_image", "nonConcurrent") {
...
```